semantic-conventions/src/tests/data/markdown/metrics_tables/http.yaml

```diff
@@ -0,0 +1,172 @@
+groups:
+  - id: http
+    prefix: http
+    brief: 'This document defines semantic conventions for HTTP client and server Spans.'
+    note: >
+        These conventions can be used for http and https schemes
+        and various HTTP versions like 1.1, 2 and SPDY.
+    attributes:
+      - id: method
+        type: string
+        required: always
+        brief: 'HTTP request method.'
+        sampling_relevant: true
+        examples: ["GET", "POST", "HEAD"]
+      - id: url
+        type: string
+        brief: >
+            Full HTTP request URL in the form `scheme://host[:port]/path?query[#fragment]`.
+            Usually the fragment is not transmitted over HTTP, but if it is known, it should be included nevertheless.
+        note: >
+          `http.url` MUST NOT contain credentials passed via URL in form of `https://username:password@www.example.com/`.
+          In such case the attribute's value should be `https://www.example.com/`.
+        sampling_relevant: true
+        examples: ['https://www.foo.bar/search?q=OpenTelemetry#SemConv']
+      - id: target
+        type: string
+        brief: 'The full request target as passed in a HTTP request line or equivalent.'
+        sampling_relevant: true
+        examples: ['/path/12314/?q=ddds#123']
+      - id: host
+        type: string
+        brief: >
+            The value of the [HTTP host header](https://tools.ietf.org/html/rfc7230#section-5.4).
+            An empty Host header should also be reported, see note.
+        note: >
+          When the header is present but empty the attribute SHOULD be set to
+          the empty string. Note that this is a valid situation that is expected
+          in certain cases, according the aforementioned
+          [section of RFC 7230](https://tools.ietf.org/html/rfc7230#section-5.4).
+          When the header is not set the attribute MUST NOT be set.
+        sampling_relevant: true
+        examples: ['www.example.org']
+      - id: scheme
+        type: string
+        brief: 'The URI scheme identifying the used protocol.'
+        sampling_relevant: true
+        examples: ["http", "https"]
+      - id: status_code
+        type: int
+        required:
+          conditional: If and only if one was received/sent.
+        brief: '[HTTP response status code](https://tools.ietf.org/html/rfc7231#section-6).'
+        examples: [200]
+      - id: flavor
+        type:
+          # Default value: `true`. If false, it helps the code gen tool to
+          # encode checks that only accept the listed values.
+          allow_custom_values: true
+          members:
```

Comment:
I know that most tests have this copy&pasta, but for new tests, maybe try to keep the YAML files minimal for what you want to test.

Reply:
Ok, I've kept it as minimal as I can for now. I've still had to duplicate `http.yaml` once, because the main one is outdated, and I wanted to use the latest one from the spec. If I update it to the latest ones lots of other tests break.

I had to add a few lines of code to specify extra files (rather than directories) to achieve this too. Other tests can follow suit in later PRs if we want to trim the copy+pasta.

Reply:
Fine with me, though long-term I'd love to have only foo/bar-style actually minimal YAML in these tests (we could have dedicated integration tests with spec YAML, maybe even pulling from spec repo), but that is a separate issue.